bw2io/backup.py

```diff
@@ -13,20 +13,23 @@
 
 
 def backup_data_directory(
-    timestamp: Optional[bool] = True, dir_backup: Optional[Union[str, Path]] = None
+    dir_backup: Optional[Union[str, Path]] = None,
+    timestamp: Optional[bool] = True,
+    file_suffix: Optional[str] = None,
 ):
     """
     Backup the Brightway2 data directory to a `.tar.gz` (compressed tar archive) in a specified directory, or in the user's home directory by default.
 
-    The file name is of the form "brightway2-data-backup.{timestamp}.tar.gz", unless `timestamp` is False, in which case the file name is "brightway2-data-backup.tar.gz".
```

Comment:
Instead of extra filename parts, why not allow users to set the filename itself? This seems more reasonable than "brightway2-data-backup{file_suffix}{timestamp}.tar.gz", which is a mouthful.